spec/controllers/orders_controller_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 describe  OrdersController  do
-  describe "index'" do  	
+  describe "Get index" do  	
```

Comment:
By the way, are you sure that **anyone** should have an access to the list of all the orders?